plugins/services/src/js/utils/ServiceConfigDisplayUtil.js

```diff
@@ -0,0 +1,691 @@
+import classNames from 'classnames';
+import React from 'react';
+import {Table} from 'reactjs-components';
+
+import HostUtil from './HostUtil';
+import Networking from '../../../../../src/js/constants/Networking';
+import Units from '../../../../../src/js/utils/Units';
+import Util from '../../../../../src/js/utils/Util';
+
+const renderMillisecondsFromSeconds = (prop, row) => {
+  let value = row[prop];
+
+  if (value != null) {
+    value = `${value * 1000} ms`;
```

Comment:
Yeah, good point. The designs showed the number in milliseconds, but I remember having a discussion about being consistent with times in the app. So I've implemented `DateUtil.getDuration` here:

![](https://cl.ly/2U3j242G392H/Screen%20Shot%202016-11-08%20at%201.19.04%20PM.png)